README.md

```diff
@@ -10,7 +10,6 @@ The following folders can be found in this repository:
 - **[`iris`](https://archive.ics.uci.edu/ml/datasets/iris)** - The iris flower data set from Ronald Fisher's 1936 paper
```

Comment:
I'm confused.  This looks like it backs out the stuff that got committed, and it adds LFS support for parquet.  All of which is good.  The part I'm confused about is taxi.parquet.  It is used in an example case.  Help me understand why this was deleted, instead of being turned into a dataset (aka folder) in the examples repo.